doc/rvv-intrinsic-spec.adoc

```diff
@@ -316,202 +351,362 @@ The following intrinsics have an irregular naming pattern.
 
-- Intrinsics with input arguments are all scalar types and scalar types alone (e.g. Vector load instruction intrinsics, Vector move instruction intrinsics)
-- Intrinsics without any input argument (e.g. `vmclr`, `vmset`, `vid`)
+- Intrinsics with input arguments that are all scalar types and scalar types alone (e.g. unmasked vector load instruction intrinsics, `vmv.s.x`)
+- Intrinsics with `vl` as the only argument (e.g. `vmclr`, `vmset`, `vid`)
 - Intrinsics with vector boolean input(s), returning a vector non-boolean vector type (e.g. `viota`)
 
-[[chapter-type-system]]
+[[type-system]]
 == Type system
 
-The RVV intrinsics are designed to be strongly-typed. The intrinsics provide `vreinterpret` intrinsics to help users go across the strongly-typed scheme if necessary.
+The intrinsics are designed to be strongly-typed. The intrinsics provide `vreinterpret` intrinsics to help users go across the strongly-typed scheme if necessary.
 
 Non-mask (integer and floating-point) data types have SEW and LMUL encoded. 
 
 [[integer-type]]
 === Integer types
 
-The integer types have EEW and EMUL encoded into the type. The first row describes the EMUL and the first column describes the data type and element width of the scalar type.
+Integer types have EEW and EMUL encoded into the type. The first row describes the EMUL and the first column describes the data type and element width of the scalar type.
 
-Type with bold font is only available when `ELEN >= 64` (that is, unavailable under `Zve32*`).
+Type with bold font is only available when `ELEN >= 64` (that is, unavailable under `Zve32*` and require at least `Zve64x`).
 
 .Integer types
-[cols="2,2,2,2,2,2,2,2"]
-[%autowidth]
+[options="autowidth,header",float="center",align="center",cols="<1,<2,<2,<2,<2,<2,<2,<2"]
 |===
-| Types     | EMUL=1/8      | EMUL=1/4       | EMUL=1/ 2      | EMUL=1        | EMUL=2        | EMUL=4        | EMUL=8
-| int8_t    | *vint8mf8_t*  | vint8mf4_t     | vint8mf2_t     | vint8m1_t     | vint8m2_t     | vint8m4_t     | vint8m8_t
-| int16_t   | N/A           | *vint16mf4_t*  | vint16mf2_t    | vint16m1_t    | vint16m2_t    | vint16m4_t    | vint16m16_t
-| int32_t   | N/A           | N/A            | *vint32mf2_t*  | vint32m1_t    | vint32m2_t    | vint32m4_t    | vint32m32_t
-| int64_t   | N/A           | N/A            | N/A            | *vint64m1_t*  | *vint64m2_t*  | *vint64m4_t*  | *vint64m8_t*
-| uint8_t   | *vuint8mf8_t* | vuint8mf4_t    | vuint8mf2_t    | vuint8m1_t    | vuint8m2_t    | vuint8m4_t    | vuint8m8_t
-| uint16_t  | N/A           | *vuint16mf4_t* | vuint16mf2_t   | vuint16m1_t   | vuint16m2_t   | vuint16m4_t   | vuint16m8_t
-| uint32_t  | N/A           | N/A            | *vuint32mf2_t* | vuint32m1_t   | vuint32m2_t   | vuint32m4_t   | vuint32m8_t
-| uint64_t  | N/A           | N/A            | N/A            | *vuint64m1_t* | *vuint64m2_t* | *vuint64m4_t* | *vuint64m8_t*
+| Types | EMUL=1/8 | EMUL=1/4 | EMUL=1/ 2 | EMUL=1 | EMUL=2 | EMUL=4 | EMUL=8
```

Comment:
In the table, I wonder if rather than using bold face for the types unavailable under Zve32* we could use a small marker such as ᵃ (e.g. maybe using a superscript `^a^`) after the type and then explain after the table that those types are not available in Zve32.

Bold face makes those type stand out too much but maybe this is not easy to achieve in AsciiDoc. In such case feel free to ignore my suggestion :)